stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -90,6 +90,11 @@ bimg  = randn(n,2)/2
                 # test empty cases
                 @test nullspace(zeros(n, 0)) == Matrix(I, 0, 0)
                 @test nullspace(zeros(0, n)) == Matrix(I, n, n)
+                # test vector cases
+                @test size(nullspace(a[:, 1])') == (n, n - 1)
+                @test size(nullspace(b[1, :])') == (n, n - 1)
+                @test size(transpose(nullspace(a[:, 1]))) == (n, n - 1)
+                @test size(transpose(nullspace(b[1, :]))) == (n, n - 1)
```

Comment:
I believe the tests should have been
```suggestion
                @test size(nullspace(a[:, 1]')) == (n, n - 1)
                @test size(nullspace(b[1, :]')) == (2, 1)
                @test size(nullspace(transpose(a[:, 1]))) == (n, n - 1)
                @test size(nullspace(transpose(b[1, :]))) == (2, 1)
```
With these modifications, they pass on current master, so shall we make this PR add only those tests, without the modifications to the code above?